lib/__tests__/Configuration-test.js

```diff
@@ -355,6 +355,103 @@ describe('Configuration', () => {
         ]);
       });
     });
+
+    describe('in multiple environments', () => {
+      beforeEach(() => {
+        FileUtils.__setFile(
+          path.join(process.cwd(), '.importjs.json'),
+          { environments: ['node', 'meteor'] });
+      });
+
+      it('returns core modules from all environments', () => {
+        const coreModules = new Configuration().get('coreModules');
+        expect(coreModules).toContain('child_process');
+        expect(coreModules).toContain('meteor/check');
+      });
+    });
+  });
+
+  describe('#get aliases', () => {
+    beforeEach(() => {
+      // TODO: replace the array-style configuration here when we have an
+      // environment that specify aliases.
+      FileUtils.__setFile(
+        path.join(process.cwd(), '.importjs.json'),
+        [
+          {
+            aliases: {
+              foo: 'foosball',
+            },
+          },
+          {
+            aliases: {
+              bar: 'barsketball',
+            },
+          },
+          {
+            aliases: () => ({
+              baz: 'bazeball',
+            }),
+          },
+        ]);
+    });
+
+    it('merges all aliases into a single object', () => {
+      expect(new Configuration().get('aliases')).toEqual({
+        foo: 'foosball',
+        bar: 'barsketball',
+        baz: 'bazeball',
+      });
+    });
+  });
+
+  describe('#get namedExports', () => {
+    beforeEach(() => {
+      // TODO: replace the array-style configuration here when we have an
+      // environment that specify namedExports.
```

Comment:
Yeah, I was on the fence about these. The reasoning is that once we no longer have local configuration through an array, we can't test configuration merging this way. The meteor config will soon have a namedExports config, and at that time we should start using it instead of this workaround.